test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -269,5 +265,24 @@ private static string GetIdentifier(string version, string type)
         {
             return $"{version}-{type}-{DateTime.Now.ToFileTime()}";
         }
+
+        private static string GetRid(ImageDescriptor imageDescriptor)
+        {
+            string rid;
+            if (imageDescriptor.IsArm)
+            {
+                rid = "linux-arm";
```

Comment:
(This existed before this change)

What is the reasoning for using `linux-arm` (i.e. a 'portable' RID) for arm architectures, but using `debian.8-x64` (i.e. a 'non-portable' RID) for x64?  It seems like it would be consistent to always use the portable RID or always use the non-portable RID.

Is it to get more test coverage?

Reply:
The reason for it is legacy related - the same tests run over all our image variants (1.* and 2.*).  Arm images exist only for 2.*.  1.* obviously doesn't support 'portable' RIDs.

That being said, there is no reason why we can't add conditional logic to use a 'portable' rid where supported.  I will log an issue for this.

Reply:
https://github.com/dotnet/dotnet-docker/issues/333

Reply:
Ah, that makes sense to me.  Thanks for the explanation.